### PR TITLE
integration-guide-feedback

### PR DESCRIPTION
**Pull request by janainawittner**
Comments directly within the text marked by [comment]: # (MY COMMENT)

### CODE REVIEW

File: content/developers/en/integration-guide.md [hunk 1]
```diff
@@ -24,14 +24,18 @@ First of all, we are very excited about the new integration you are going to dev
 
 When you are ready to start, please let us know at integration@hubrise.com. We won't just wait and see while you do the work, but we will actively help you during the process. Here is what HubRise can do for you:
 
+[comment]: # (WHEN WE KNOW, WE CAN GIVE THEM FREE ACCESS TO THE TEST ACCOUNT ALSO, IT AVOIDS LATE MINUTE REQUESTS TO SPOT AUTOMATED MESSAGES FOR DUE SUBSCRIPTION TO BE SENT)
```
Comment (atorin) [inline content/developers/en/integration-guide.md:27]: OK! I've rephrased the above paragraph to take this into account

File: content/developers/en/integration-guide.md [hunk 2]
```diff
@@ -24,14 +24,18 @@ First of all, we are very excited about the new integration you are going to dev
...
 - We write a report with our recommendations.
 
 ### Assessing the Integration
 
-When your integration is ready, we review it with you and fill the Integration Sheet together. In this way, we know exactly how your integration works, and we can document it thoroughly.
+When your integration is ready, we review it with you and fill an Integration Sheet together. In this way, we know exactly how your integration works, and we can document it thoroughly.
+[comment]: # (YOU HAVE NOT EXPLAINED WHAT AN INTEGRATION SHEET IS, YET YOU REFER TO IT AS IF IT WAS SOMETHING KNOWN. SHOULD IT NOT BE PRESENTED FIRST? ALSO I WOULD REITERATE THAT WE WILL PROVIDE WRITTEN FEEDBACK. SOMETHING ALONG THE LINES OF : When your integration is ready, we review it with you. To do this we will check the logs on test transaction you have done and ask you a few questions. The review process is structure around an Integration Sheet. To see what to expect, check our [Integration Sheet template](https://docs.google.com/spreadsheets/d/1df-QRlD9h8M58bpFoFaCEzU5pbmYSeHXOLqIVip9-5s/edit?usp=sharing).
```
Comment (atorin) [inline content/developers/en/integration-guide.md:37]: OK thanks! I've used your suggestion with few minor changes.

File: content/developers/en/integration-guide.md [hunk 3]
```diff
@@ -45,6 +49,7 @@ We document integrations to:
 - Provide support teams with links they can include in support emails, instead of repeating common explanations multiple times.
 - Build a public knowledge base that everyone can view.
 - Improve transparency across the various integrated solutions in the HubRise ecosystem.
+[comment]: # (I THINK THIS SHOULD BE ON BULLET ONE, IT'S A VERY IMPORTANT POINT :-)
```
Comment (atorin) [inline content/developers/en/integration-guide.md:52]: OK!

File: content/developers/en/integration-guide.md [hunk 4]
```diff
@@ -3,8 +3,8 @@ title: Integration Guide
 position: 3
 layout: documentation
 meta:
-  title:
-  description:
+  title: HubRise Integration Guide
+  description: An integration guide with best practices and practical suggestions to develop an integration with HubRise.
```
Comment (janainawittner) [inline content/developers/en/integration-guide.md:7]: We need more words I think...
Could say something like: An integration guide with best practices and practical suggestions to develop an integration with HubRise. Information on how HubRise will help.

Remember: https://trello.com/c/OHzVRlYD/59-meta-title-description

File: content/developers/en/integration-guide.md [hunk 5]
```diff
@@ -22,7 +22,10 @@ It is composed of the following sections:
 
 First of all, we are very excited about the new integration you are going to develop!
 
-When you are ready to start, please let us know at integration@hubrise.com. We won't just wait and see while you do the work, but we will actively help you during the process. Here is what HubRise can do for you:
+When you are ready to start, please let us know at integration@hubrise.com.
+We will waive your subscription fee so that you can place unlimited test orders during the development phase. (Free accounts are limited to 50 orders per month.)
```
Comment (janainawittner) [inline content/developers/en/integration-guide.md:26]: Remove this I think : (Free accounts are limited to 50 orders per month.)
In the style guide we say people don't read what is between brakets. It comes from the Google Style Guide :-)

File: content/developers/en/integration-guide.md (hunk 4 above)
Comment (janainawittner) [inline content/developers/en/integration-guide.md:7]: We need more words I think...
Could say something like: An integration guide with best practices and practical suggestions to develop an integration with HubRise. Information on how HubRise will help.

Remember: https://trello.com/c/OHzVRlYD/59-meta-title-description

File: content/developers/en/integration-guide.md (hunk 5 above)
Comment (antmonnier) [reply content/developers/en/integration-guide.md]: Done

File: content/developers/en/integration-guide.md (hunk 4 above)
Comment (antmonnier) [reply content/developers/en/integration-guide.md:7]: Was not too sure what to add. Descriptions will need to be reviewed at some point.

**Comment by antmonnier** [comment]
@janainawittner I've moved your comments to a separate issue: https://github.com/HubRise/website/issues/169

**Pull request merged**